controllers/controllers.articles.js

```diff
@@ -1,11 +1,26 @@
-const { selectArticles } = require("../models/models.articles");
+const {
+  selectArticles,
+  selectArticleById,
+} = require("../models/models.articles");
 
 exports.getArticles = (req, res, next) => {
   selectArticles()
-    .then(({ rows }) => {
+    .then((rows) => {
       res.status(200).send({ articles: rows });
     })
     .catch((err) => {
       next(err);
     });
 };
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+
+  selectArticleById(article_id)
+    .then((rows) => {
+      res.status(200).send({ article: rows });
```

Comment:
as you are already returning `rows` deconstructed in the model - you can declare the argument in the .then to be whatever name you want - if you declare it as `article` then you only need to send `({ article });` in your response object